cairo_vm_hints/src/hints/lib/verifiers/evm/account_verifier.rs

```diff
@@ -0,0 +1,230 @@
+use cairo_vm::{
+    hint_processor::builtin_hint_processor::{
+        builtin_hint_processor_definition::HintProcessorData,
+        hint_utils::{get_integer_from_var_name, get_ptr_from_var_name, insert_value_from_var_name, insert_value_into_ap},
+    },
+    types::{exec_scope::ExecutionScopes, relocatable::MaybeRelocatable},
+    vm::{errors::hint_errors::HintError, vm_core::VirtualMachine},
+    Felt252,
+};
+use num_bigint::BigUint;
+use num_traits::Num;
+use std::collections::HashMap;
+
+use crate::{
+    hint_processor::models::proofs::Proofs,
+    hints::{
+        lib::utils::{count_leading_zero_nibbles_from_hex, split_128},
+        vars,
+    },
+};
+
+pub const HINT_BATCH_ACCOUNTS_LEN: &str = "len(batch[\"accounts\"])";
```

Comment:
this hint is not present in the code

Reply:
we have 
```
local n_accounts: felt = nondet %{ len(batch.accounts) %};
```

Reply:
Should the hint be "len(batch.accounts)" instead?

Reply:
it is `nondet` keyword so it will be compiled to the 
memory[ap] = to_felt_or_relocatable(len(batch.accounts))